static/js/typing_data.js

```diff
@@ -64,20 +82,40 @@ export function get_all_pms_typists() {
     return pms_typists;
 }
 
+export function get_stream_typists(stream_id, topic) {
+    return typist_dct.get(JSON.stringify({stream_id, topic})) || [];
+}
+
 // The next functions aren't pure data, but it is easy
 // enough to mock the setTimeout/clearTimeout functions.
-export function clear_inbound_timer(group) {
-    const key = get_key(group);
+function clear_inbound_timer(key) {
     const timer = inbound_timer_dict.get(key);
     if (timer) {
         clearTimeout(timer);
         inbound_timer_dict.set(key, undefined);
     }
 }
 
-export function kickstart_inbound_timer(group, delay, callback) {
+export function clear_pms_inbound_timer(group) {
     const key = get_key(group);
-    clear_inbound_timer(group);
+    clear_inbound_timer(key);
+}
+
+export function clear_streams_inbound_timer(stream_id, topic) {
+    const key = JSON.stringify({stream_id, topic});
```

Comment:
We're going to want this to be a `get_topic_key(stream_id, topic)` -- topics are case-insensitive, and we'll want to implement that in only one place.

Reply:
Added a `get_topic_key`.